src/index.js

```diff
@@ -10,79 +10,87 @@ const {
 const escape = require('html-escape');
 const fs = require('fs');
 const path = require('path');
+const readline = require('readline');
 
 module.exports = {
   generate: argv => {
     const templateFilePath = path.join(__dirname, "template", "element-doc-template.html");
-    const itemsRootPath = path.resolve(argv[2] || './');
+    const args = {
+      root: path.resolve("./"),
+      overwrite: argv[2] === "-f"
+    };
 
     const analyzer = new Analyzer({
-      urlLoader   : new FSUrlLoader(itemsRootPath),
+      urlLoader   : new FSUrlLoader(args.root),
       urlResolver : new PackageUrlResolver()
     });
 
     analyzer.analyzePackage().then(_package => {
       const metadata = generateAnalysis(_package, '');
-      let elements;
 
-      const processingPackage = !Boolean(
-          metadata.elements.find(item => item.path.indexOf(path.sep) === -1));
+      console.log(`Processing element from the ${args.root} directory`);
```

Comment:
It seems to be a popular practice to end such "progress-initiating" messages with an ellipsis. E.g. `git clone https://... .`
prints out a message like this `Cloning into '.'...`